examples/nginx-access-logs-streaming/README.md

```diff
@@ -0,0 +1,640 @@
+Removing CSV headers from /mnt/csv/nginx/access.log.11.csv.
+Removing CSV headers from /mnt/csv/nginx/access.log.12.csv.
+Removing CSV headers from /mnt/csv/nginx/access.log.13.csv.
+Removing CSV headers from /mnt/csv/nginx/access.log.14.csv.
+Removing CSV headers from /mnt/csv/nginx/access.log.2.csv.
+Removing CSV headers from /mnt/csv/nginx/access.log.3.csv.
+Removing CSV headers from /mnt/csv/nginx/access.log.4.csv.
+Removing CSV headers from /mnt/csv/nginx/access.log.5.csv.
+Removing CSV headers from /mnt/csv/nginx/access.log.6.csv.
+Removing CSV headers from /mnt/csv/nginx/access.log.7.csv.
+Removing CSV headers from /mnt/csv/nginx/access.log.8.csv.
+Removing CSV headers from /mnt/csv/nginx/access.log.9.csv.
+Combined all access logs into a single file: /mnt/csv/nginx/access.log.csv.
+removed '/mnt/csv/nginx/access.log.0.csv'
+removed '/mnt/csv/nginx/access.log.1.csv'
+removed '/mnt/csv/nginx/access.log.2.csv'
+removed '/mnt/csv/nginx/access.log.3.csv'
+removed '/mnt/csv/nginx/access.log.4.csv'
+removed '/mnt/csv/nginx/access.log.5.csv'
+removed '/mnt/csv/nginx/access.log.6.csv'
+removed '/mnt/csv/nginx/access.log.7.csv'
+removed '/mnt/csv/nginx/access.log.8.csv'
+removed '/mnt/csv/nginx/access.log.9.csv'
+removed '/mnt/csv/nginx/access.log.10.csv'
+removed '/mnt/csv/nginx/access.log.11.csv'
+removed '/mnt/csv/nginx/access.log.12.csv'
+removed '/mnt/csv/nginx/access.log.13.csv'
+removed '/mnt/csv/nginx/access.log.14.csv'
+Extracting only IP addresses from /mnt/csv/nginx/access.log.csv to /mnt/csv/nginx/access.ipinfo.tmp ...
+Removing duplicate IP addresses from /mnt/csv/nginx/access.ipinfo.tmp ...
+Combined all IP addresses into a single file: /mnt/csv/nginx/access.ipinfo.
+removed '/mnt/csv/nginx/access.ipinfo.tmp'
+Geo-locating all the IP addresses in bulk using the IPInfo API (https://ipinfo.io):
+   Total IP addresses that will be looked up in bulk using /mnt/csv/nginx/access.ipinfo: 3037.
+   Geo-lookup of 3037 IP addresses written to file: /mnt/csv/nginx/access.ipinfo.csv.
+   Complete!
+removed '/mnt/csv/nginx/access.ipinfo'
+</pre>
+</details>
+
+5. Next copy over the generated CSV files to `proton-data/user_files`
+```bash
+cp -v /mnt/nginx/csv/access.* proton-data/user_files/
+'/mnt/nginx/csv/access.ipinfo.csv' -> 'proton-data/user_files/access.ipinfo.csv'
+'/mnt/nginx/csv/access.log.csv' -> 'proton-data/user_files/access.log.csv'
+```
+
+6. Run the additional [sql scripts](scripts/sql) which will load the generated CSV files into Timeplus Proton:
+```bash
+./proton client --host 127.0.0.1 --multiquery < sql/01_create-tables.sql
+./proton client --host 127.0.0.1 --multiquery < sql/02_csv-import.sql
+
+# this alternative command will also work
+cat sql/<file-name>.sql | ./proton client --host 127.0.0.1 --multiquery
+```
+The ingestion of both files: `access.log.csv` and `access.ipinfo.csv` was pretty fast with each import completing under a second on my machine: 
+```bash
+-- access.log.csv
```

Comment:
suggest you copy the line 391 here, to make it as full bash script. Same as the next file

Reply:
Could you please clarify what you mean here?

Reply:
Hi @ayewo , when I browsed the blog/README and saw this
```bash
-- access.log.csv
Query id: 711127ab-132c-4381-bdbd-0d109b958cea
```
It'll be a bit confusing. Probably put the full command there, e.g.
```bash
cat sql/access.log.csv.sql | ./proton client --host 127.0.0.1 --multiquery
Query id: 711127ab-132c-4381-bdbd-0d109b958cea

Ok.

0 rows in set. Elapsed: 0.128 sec. Processed 65.25 thousand rows, 16.25 MB (508.14 thousand rows/s., 126.51 MB/s.)

```

Reply:
Should look better now.